src/libsyntax/feature_gate.rs

```diff
@@ -302,7 +305,6 @@ declare_features! (
     (removed, struct_inherit, "1.0.0", None),
     (removed, test_removed_feature, "1.0.0", None),
     (removed, visible_private_types, "1.0.0", None),
-    (removed, unsafe_no_drop_flag, "1.0.0", None)
```

Comment:
FWIW library features do get removed all the time so I don't think we need to bother anymore with this list.